k8sPlugin/Jenkinsfile

```diff
@@ -1,8 +1,8 @@
 //Inspired by: https://github.com/jenkinsci/kubernetes-plugin/blob/master/README.md#container-group-support
 podTemplate(containers: [
-    containerTemplate(name: 'maven', image: 'maven:3.6.2-jdk-8', ttyEnabled: true, command: 'cat'),
-    containerTemplate(name: 'golang', image: 'golang:1.13.3', ttyEnabled: true, command: 'cat')
-  ]) {
+    containerTemplate(name: 'maven', image: 'maven:3.6.2-jdk-8', ttyEnabled: true, command: 'cat', runAsUser: 1000),
```

Comment:
It would be better to set `runAsUser: 1000` one level up. (If possbile).
That way, it would also affect the jnlp container.

Reply:
See: https://github.com/jenkinsci/kubernetes-plugin#pod-and-container-template-configuration

Reply:
With "one level up" you mean the Jenkins Global configuration? We need to check this. This would be here:
https://github.com/SAP/stewardci-jenkinsfilerunner-image/blob/master/jenkinsfile-runner-base-image/casc.yml

Reply:
No, I mean in the podTempalte. Jenkins Global configuration is at least five levels up.

Reply:
Ah, OK